src/locales/en/translation.json

```diff
@@ -301,17 +301,17 @@
+    "faqAnswer2": "In celebration of the Winter Solstice, there will be an all-day event in Chinatown to honor the start of the season and a “Light Up Chinatown” Ceremony. During the celebration, all are welcome to enjoy delicious food and holiday shopping. Invitations with exact details will be sent to those who donated $150+ to our Light Up Chinatown Fundraising Campaign. The lights will be installed and available for viewing on December 23th.",
     "faqQuestion3": "How will the light fixtures and lanterns be installed for “Light Up Chinatown”?",
     "faqAnswer3": "<0>Alternative Earth Care</0> has a four man team that will install 14 sections of lighting on Mott Street from Canal to Bayard with lanterns hung between each light bulb for Light Up Chinatown.",
     "faqQuestion4": "I donated $150+ before December 15th, where can I pick up my personalized lantern?",
-    "faqAnswer4": "The pick up location for lanterns will be at Pearl River Mart on 395 Broadway, New York, NY 10013 after 12/26 anytime from 11am - 6pm. You should have received an email confirmation with detailed instructions on pick up. If you provided your shipping address, we will ship you your personalized lantern after December 21st.",
+    "faqAnswer4": "The pick up location for lanterns will be at Pearl River Mart on 395 Broadway, New York, NY 10013 after December 26th anytime from 11am - 6pm. You should have received an email confirmation with detailed instructions on pick up. If you provided your shipping address, we will ship you your personalized lantern after December 21st.",
     "faqQuestion5": "I have not received an email confirmation. Who can I contact?",
     "faqAnswer5": "Please contact us at <0>hello@sendchinatownlove.com</0> with the subject line “Light Up Chinatown”.",
     "faqQuestion6": "I love Chinatown and I’d love to further support its businesses. Where can I do that?",
     "faqAnswer6": "Thank you for sharing your love with Chinatown! To continue supporting this community, check out our <0>merchant page</0> for the full list of Asian-owned businesses we’re working with. From here, you can make a donation or buy a gift card.",
     "faqQuestion7": "What is Send Chinatown Love?",
-    "faqAnswer7": "Send Chinatown Love is a New York-based, entirely volunteer-run organization that helps the Chinatowns across the city get back on their feet The goal of Send Chinatown Love is to uplift  Asian and immigrant-owned small businesses by providing donations and personalized business development support . Check out our full list of partner merchants <>here</>.",
+    "faqAnswer7": "Send Chinatown Love is a New York-based, entirely volunteer-run organization that helps the Chinatowns across the city get back on their feet The goal of Send Chinatown Love is to uplift  Asian and immigrant-owned small businesses by providing donations and personalized business development support.",
```

Comment:
Missing a full stop/period after "get back on their  feet". 
"get back on their feet. The goal of Send Chinatown Love..."